### PR TITLE
Fix no-preview placeholder BlockIcon usage

### PR DESCRIPTION
**Pull request by notnownikki**
## Description

Update the Placeholder used to URLs that can be embedded but can't be previewed in the editor due to sandbox security settings.

## How has this been tested?

Create a new post, try to embed https://www.facebook.com/buzzfeedladylike/posts/2290362924583261

## Types of changes
Bug fix

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [x] My code has proper inline documentation.

### CODE REVIEW

**Comment by notnownikki** [comment]
Resolves: https://github.com/WordPress/gutenberg/issues/9444

**Review by jorgefilipecosta — APPROVED** [review]
Looks good to me I think we just need to move the translator comment before the merge.

File: packages/block-library/src/embed/index.js [hunk 1]
```diff
@@ -188,10 +188,10 @@ export function getEmbedEdit( title, icon ) {
 				);
 			}
 
+			const label = sprintf( __( '%s URL' ), title );
+
 			if ( ! preview || previewIsFallback || editingURL ) {
 				// translators: %s: type of embed e.g: "YouTube", "Twitter", etc. "Embed" is used when no specific type exists
```
[the remark this answers is not in the document]
Comment (notnownikki) [reply packages/block-library/src/embed/index.js]: Gah, I missed that one! Thanks :)

File: packages/block-library/src/embed/index.js [hunk 2]
```diff
@@ -237,7 +237,7 @@ export function getEmbedEdit( title, icon ) {
 				<figure className={ classnames( className, 'wp-block-embed', { 'is-video': 'video' === type } ) }>
 					{ controls }
 					{ ( cannotPreview ) ? (
-						<Placeholder icon={ icon } label={ __( 'Embed URL' ) }>
+						<Placeholder icon={ <BlockIcon icon={ icon } showColors /> } label={ label } className="wp-block-embed">
```
Comment (mtias) [inline packages/block-library/src/embed/index.js]: Why are we duplicating the `class` attribute here (it's already on the `figure` above)? cc @notnownikki @jorgefilipecosta

**Pull request merged**